internal/store/bigtable/reader.go

```diff
@@ -76,72 +76,62 @@ func readRowFn(
 }
 
 // Read reads BigTable rows from multiple Bigtable in parallel.
-//
-// Reading multiple rows is chunked as the size limit for RowSet is 500KB.
-//
-// Args:
-// baseBt: The bigtable that holds the base cache
-// rowSet: BigTable rowSet containing the row keys.
-// action: A callback function that converts the raw bytes into appropriate
-//		go struct based on the cache content.
-// getToken: A function to get back the indexed token (like place dcid) from
-//		bigtable row key.
-//
+// Note all Bigtable read use the same set of rowList.
 func Read(
 	ctx context.Context,
 	btGroup *Group,
-	rowSet cbt.RowSet,
+	rowSet cbt.RowList,
```

Comment:
Is it worthwhile to change the variable name: rowSet->rowList?

Reply:
Done